pkg/services/pluginsintegration/pluginconfig/config.go

```diff
@@ -39,6 +39,7 @@ func ProvidePluginManagementConfig(cfg *setting.Cfg, settingProvider setting.Pro
 		cfg.DisablePlugins,
 		cfg.HideAngularDeprecation,
 		cfg.ForwardHostEnvVars,
+		cfg.PluginInstallToken,
```

Comment:
I think the issue is the naming. The `install_token` in reality can be used for any request, so far it has this name because it was being used in plugin installation. I'll rename it to `GrafanaComToken` or something more generic, then I think it will make more sense to be used in all requests to GCOM API when it's available. What do you think?

Reply:
Renaming is probably fine since we are the only ones using the setting but needs to be treated carefully.

Reply:
Since I'll have to rename the property on  default.ini and change hosted-grafana to inject it, I will let this part for another PR, if you agree.

Reply:
up to you how you want to proceed 👍